src/root_files/app_2/sitemap.app_2.id.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@OskarAhl do we server the app on `.me` domain just for `id`?

Reply:
Hmm it's implemented this way for `binary.com`  🤔